pyangbind/lib/yangtypes.py

```diff
@@ -377,19 +377,25 @@ class TypedList(collections.MutableSequence):
     _list = list()
 
     def __init__(self, *args, **kwargs):
+      self._unique = kwargs.pop('unique', False)
```

Comment:
Is there ever a scenario that we expect that a list is *not* unique? If not, it might be best to make this flag work the other way around such that we have something that says to skip the uniqueness check?